slides/index.html

```diff
@@ -32,49 +44,108 @@
 # Dette kurset
 
 ## 3 timer
+
+Blåtann  
+mbed  
+Hjelpemidler
+
 ## 3 oppgaver
 
+Lag en mbed med blåtann  
+Chat fra telefonen til mbed-en  
+Something, something, third task
+
 ???
 
-Vi tar en pause mellom hver oppgave
+- Vi holder korte foredrag om teknologi fulgt av oppgaver
+- Under oppgavene går vi rundt og hjelper til, rekk opp hånda om dere trenger hjelp
+- Vi tar ca et kvarters pause mellom hver oppgave
 
 ---
 class: center
 
 # http://bftj.co
 
-*screenshot*
+![](/images/bftjco.png)
 
 ???
 
-Vi har en nettside, som inneholder guider og oppgavene etc.
+- Vi har en nettside, som inneholder **guider**, **oppgavene**, **slides** osv
+- Bruk denne som ressurs når du jobber
+- Denne er tilgjengelig også etter kurset så den kan brukes om du trenger hjelp senere
 
 ---
-class: center
+class: center, middle
 
-# Bluetooth
+# Blåtann
```

Comment:
Mention that we're doing BLE and not regular Bluetooth perhaps? ("what is the difference between gatt/service-oriented bluetooth and the regular kind that is in the douche-headset?")
![image](https://cloud.githubusercontent.com/assets/578029/6433213/3c4dbce0-c069-11e4-9adc-867f34762218.png)